apps/entry/filter_set.py

```diff
@@ -213,6 +220,11 @@ def authoring_organization_types_filter(self, qs, name, value):
             return qs.filter(organization_types__in=[ot.id for ot in value])
         return qs
 
+    def entry_ids_filter(self, qs, name, value):
+        if value:
+            return qs.filter(id__in=[entry.id for entry in value])
+        return qs
+
```

Comment:
This might not be required. We should be able to define this in the `ModelMultipleChoiceFilter`.

https://django-filter.readthedocs.io/en/stable/ref/filters.html#to-field-name
https://django-filter.readthedocs.io/en/stable/ref/filters.html#lookup-expr